config-service/build.gradle.kts

```diff
@@ -64,7 +64,7 @@ dependencies {
 
   runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.13.3")
   runtimeOnly("io.grpc:grpc-netty:1.36.0")
-
+  runtimeOnly("io.netty:netty-codec-http2:4.1.60.Final")
```

Comment:
Any reason for moving this out of `constraints` block?

Reply:
oh you mean, I can completely get rid of it, since it was added in the constraints because of the snyk issues before. And, it was coming via transitive dependencies

Reply:
moving it back to the constraints block

Reply:
moving it back to the constraints block

Reply:
> oh you mean, I can completely get rid of it, since it was added in the constraints because of the snyk issues before. And, it was coming via transitive dependencies

yes. Aaron has explained the reason here: https://github.com/hypertrace/span-normalizer/pull/32#discussion_r512809672